### PR TITLE
updated gitignore

### PR DESCRIPTION
**Pull request by SanyuktaSaha**
Added an additional information.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -24,6 +24,8 @@ install/status.json
 custom/modules/Connectors/connectors/sources/ext/*/*/config.php
 # Logs files can safely be ignored.
 *.log
+# The silent upgrade scripts aren't needed.
+silentUpgrade*.php
```
Comment (Dillon-Brown) [inline .gitignore]: Why do you feel these aren't needed? the ```silentUpgrade.php``` file is used by the ```UpgradeWizard/``` module to execute upgrades to an existing instance through the CLI.

File: HandleAjaxCall.php [hunk 2]
```diff
@@ -58,3 +58,4 @@
         echo 'no method';
     }
    // sugar_cleanup();
+?>
```
Comment (Dillon-Brown) [inline HandleAjaxCall.php]: Redundant closing tag.
```suggestion
```

File: vcal_server.php [hunk 3]
```diff
@@ -43,3 +43,4 @@
 
 require_once __DIR__ . '/include/entryPoint.php';
 require_once __DIR__ . '/modules/vCals/Server.php';
+?>
```
Comment (Dillon-Brown) [inline vcal_server.php]: Redundant closing tag.
```suggestion
```

File: .travis.yml [hunk 4]
```diff
@@ -4,6 +4,8 @@ sudo: required
 
 matrix:
   fast_finish: true
+  allow_failures:
+    - php: hhvm   # ZIP Error(11): Status(0): Arhive(cache/zipTest.zip): Directory(cache)
```
Comment (Dillon-Brown) [inline .travis.yml]: What was the reason for adding ```hhvm``` as an allowed failure? afaik we don't actually use ```hhvm``` in our Travis builds.

**Comment by samus-aran** [comment]
Please provide context to the changes you have proposed. We have identified that the changes are not bringing value to the codebase and we will mark this as invalid. I suggest you have a re-think on your approach on providing quality contributions to the project (and others) in the future. Thanks.

**Pull request closed without merging**